### PR TITLE
Feature/dashboard new events

### PR DESCRIPTION
**Pull request by sandorbanvitagroup**

### CODE REVIEW

File: src/assets/i18n/custom-translations.json [hunk 1]
```diff
@@ -127,6 +127,8 @@
   "eventGroup.linkEvent.reportDate": "Report date",
   "eventGroup.linkEvent.filters.sortBy": "Sort by",
   "eventGroup.linkEvent.filters.from": "Event from",
-  "dashboard.newCase.casesWithDefinition": "Cases with reference definotion fulfilled",
-  "dashboard.newCase.rate": "Rate"
+  "dashboard.newCase.casesWithDefinition": "Cases with reference definition fulfilled",
+  "dashboard.newCase.rate": "Rate",
+  "dashboard.newCase.newCases": "New Cases",
+  "dashboard.newEvent.newEvents": "New Events"
```
Comment (danielrus) [inline src/assets/i18n/custom-translations.json]: why add new translations when we already have them?
(both New cases and New events exist)

File: projects/api-proxy/proxy-server.js [hunk 2]
```diff
@@ -174,6 +174,17 @@ app.use('/sormas-rest/dashboard/newCases', (req, res) => {
   });
 });
 
+app.use('/sormas-rest/dashboard/newEvents', (req, res) => {
```
Comment (danielrus) [inline projects/api-proxy/proxy-server.js]: no need to hardcode the response, it works (for new cases too)

File: src/assets/i18n/custom-translations.json (hunk 1 above)
Comment (sandorbanvitagroup) [reply src/assets/i18n/custom-translations.json]: Resolved.

File: projects/api-proxy/proxy-server.js (hunk 2 above)
Comment (sandorbanvitagroup) [reply projects/api-proxy/proxy-server.js]: Resolved.

File: src/app/dashboard/components/dashboard-new-events/dashboard-new-events.component.html [hunk 3]
```diff
@@ -1 +1,43 @@
-<p>dashboard-new-events works!</p>
+<div class="dashboard-new-events" *ngIf="newEvents">
+  <h1>
+    <span class="nr-of-events">{{
+      newEvents.CLUSTER +
+        newEvents.EVENT +
+        newEvents.SIGNAL +
+        newEvents.SCREENING +
+        newEvents.DROPPED || 0
```
Comment (adriandumavitagroup) [inline src/app/dashboard/components/dashboard-new-events/dashboard-new-events.component.html]: please create a method here to be more clear

Comment (sandorbanvitagroup) [reply src/app/dashboard/components/dashboard-new-events/dashboard-new-events.component.html]: Fixed.

**Review by adriandumavitagroup — APPROVED** [review]

**Pull request merged**